### DISCUSSION TITLE
Saving Home Address in Keepassxc

### DISCUSSION THREAD

**Discussion by DeNohvo** [keepassxreboot/keepassxc#10835] [Q&A] [answered]
I have not found any guides or see any options to store address in keepassx,. How do you store address that can be auto filled by keepassxc browser extension?

I have disabled passwords and storing address or other autofill info in my browser.

**Comment by droidmonkey** [comment]
Save the parts as advanced attributes then create an autotype sequence that will type the parts out as you please. See the user guide for how to.

**Comment by DeNohvo** [comment]
I did check the user guide and found no sections on how to use or setup advanced attributes, just an example for hiding recovery code which is self-explanatory.  Can you call an attribute "Phone Number" or it has to be "Phone_Number"? 

Saving an address to keepassxc should be pretty basic and built-in without these extra step IMO.

**Reply by droidmonkey** [reply, marked as answer]
KeePass compatible products are not designed around addresses and other non-security related data. We do plan to support something like this in the future, but here is how you do it now:

![image](https://github.com/keepassxreboot/keepassxc/assets/2809491/af5130a9-54fc-4959-b4ad-7f5d2202285a)

![image](https://github.com/keepassxreboot/keepassxc/assets/2809491/d2ec3e15-52aa-4e39-b462-a3407ccb5d2c)

![image](https://github.com/keepassxreboot/keepassxc/assets/2809491/21a98c72-94d2-46d7-8cac-d2be69a983d2)

![image](https://github.com/keepassxreboot/keepassxc/assets/2809491/295a4159-27df-44db-a717-bab72e11dd01)


Relevant docs:

https://keepassxc.org/docs/KeePassXC_UserGuide#_additional_attributes
https://keepassxc.org/docs/KeePassXC_UserGuide#_auto_type
